global.d.ts

```diff
@@ -0,0 +1,3 @@
+declare module 'node:utils' {
+  export function styleText(format: string, text: string): string
```

Comment:
Let’s also add comment that it is a temporary hack. Something like:

```js
  // Hack until this PR will be merged
  // https://github.com/DefinitelyTyped/DefinitelyTyped/pull/69212
  export function styleText(format: string, text: string): string
```

Those comments help to clean hacks when you don’t need them anymore. Without them, it is easy to find yourself year later with hack 100+ hacks when you don’t know what you can delete now.